### PR TITLE
Th related products

### PR DESCRIPTION
**Pull request by thomHayner**
Fix Modals re: basic functioning (no styling)
Link Modals to ProductCard button
Various Fixes to Small Problems

### CODE REVIEW

File: src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx [hunk 1]
```diff
@@ -63,9 +62,17 @@ class Carousel extends Component {
     }
 
     return (
-      <div className="container-fluid carousel-main">
+      <div className="container-fluid rp-carousel-main">
         <div className="row">
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx:68]: Expanded for Readability

File: src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -84,7 +92,15 @@ class Carousel extends Component {
             </div>
 
           </div>
-          <div className="col-xs-1 col-sm-1 carousel-arrow" onClick={() => this.next('right')} onKeyPress={() => {}} role="button" tabIndex={0}><div>&#9654;</div></div>
+          <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx:99]: Expanded for Readability

File: src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx [hunk 3]
```diff
@@ -75,6 +82,7 @@ class Carousel extends Component {
                     <ProductCard
                       key={i}
                       carouselType="suggestions"
+                      buttonType="+"
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx:91]: buttonTypre Prop Added

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 4]
```diff
@@ -61,9 +67,17 @@ class MyOutfitCarousel extends Component {
     }
 
     return (
-      <div className="container-fluid carousel-main">
+      <div className="container-fluid rp-carousel-main">
         <div className="row">
-          <div className="col-xs-1 col-sm-1 carousel-arrow" onClick={() => this.next('left')} onKeyPress={() => {}} role="button" tabIndex={0}><div>&#9664;</div></div>
+          <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:72]: Expanded for Readability

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 5]
```diff
@@ -72,22 +86,33 @@ class MyOutfitCarousel extends Component {
                   <OutfitCard
                     currentProduct={this.props.currentProduct}
                     cardIndex={this.state.myOutfitCarouselIndex}
-                    onClick={() => this.addToOutfit(this.props.currentProduct)}
+                    handleClick={() => this.addToOutfit()}
                   />
                 </div>
                 {showHand.map((i, id) => (
                   <ProductCard
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:93]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 6]
```diff
@@ -72,22 +86,33 @@ class MyOutfitCarousel extends Component {
...
+                    buttonType="x"
+                    handleDelete={this.removeFromOutfit}
                   />
                 ))}
               </div>
             </div>
 
           </div>
-          <div className="col-xs-1 col-sm-1 carousel-arrow" onClick={() => this.next('right')} onKeyPress={() => {}} role="button" tabIndex={0}><div>&#9654;</div></div>
+          <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:107]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx [hunk 7]
```diff
@@ -20,65 +22,41 @@ const ProductCard = (props) => {
...
-                {card.default_price}
-                .00
-              </div>
-              <div className="row product-card-star">
-                <Stars />
-              </div>
-            </div>
-          </div>
+
+  return (
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx:26]: The Rest Of This File Is All Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 8]
```diff
@@ -1,29 +1,75 @@
 /* eslint-disable no-undef */
 import React, { useState, useEffect } from 'react';
 import './ProductCardImage.scss';
+import ModalDashboard from '../modal/ModalDashboard/ModalDashboard';
+// import helperApi from '../../../../Shared/api';
 
-const ProductCardImage = (props) => {
+const ProductCardImage = ({
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:7]: Destructuring

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 9]
```diff
@@ -1,29 +1,75 @@
...
       .then((data) => setImageUrl(data.results[0].photos[0].thumbnail_url))
       .catch((err) => err);
   });
 
   if (imageUrl == null) {
     return (
-      <div className="card-image">
-        <button type="button" className="button-overlay"> +</button>
-        <img src="https://www.warnersstellian.com/Content/images/product_image_not_available.png" alt="ALT_IMAGE_TEXT_HERE" />
+      <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:22]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 10]
```diff
@@ -4,31 +4,57 @@
...
       <div className="modal-dashboard">
-        <ModalPopUp show={this.state.showRelatedProductsModal} handleChangeVisibility={this.changeModalVisibility}>
+        <CompareModalPopUp show={this.state.showSuggestionsModal} handleChangeVisibility={this.changeModalVisibility}>
           <p>MODAL</p>
           <p>DATA AND STUFF</p>
-        </ModalPopUp>
-        <div className="modal-show-button" onClick={this.changeModalVisibility}>CLICK HERE TO OPEN COMPARE MODAL</div>
+        </CompareModalPopUp>
+        {/* <div className="container-fluid"> */}
+        <MyOutfitModalPopUp
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx:74]: Expanded for Readability, Code Cleanup

File: src/Components/RelatedProducts/RelatedProducts.jsx [hunk 11]
```diff
@@ -1,9 +1,10 @@
 /* eslint-disable react/destructuring-assignment */
 import React, { Component } from 'react';
+// import { Connect } from 'react-redux';
 import './RelatedProducts.scss';
 import Carousel from './RPComponents/Carousel/Carousel';
 import MyOutfitCarousel from './RPComponents/MyOutfitCarousel/MyOutfitCarousel';
-import ModalDashboard from './RPComponents/modal/ModalDashboard/ModalDashboard';
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RelatedProducts.jsx:6]: Code CleanUp

File: src/Components/RelatedProducts/RPComponents/modal/SuggestionsPopUp/SuggestionsModalPopUp.scss [hunk 12]
```diff
@@ -1,4 +1,4 @@
-.modal {
+.rp-modal {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/SuggestionsPopUp/SuggestionsModalPopUp.scss]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.scss [hunk 13]
```diff
@@ -1,3 +1,10 @@
-.modal-show-button {
+.rp-modal-my-outfit-are-you-sure-delete-text {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 14]
```diff
@@ -4,31 +4,57 @@
...
+          className="rp-modal-my-outfit-are-you-sure-delete-text"
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx:75]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 15]
```diff
@@ -4,31 +4,57 @@
...
+          show={this.state.showMyOutfitModal}
+          handleConfirmVisibility={this.confirmedDeletionFromMyOutfit}
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx:77]: Separate The My Outfit Delete Options (Confirm / Cancel)

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 16]
```diff
@@ -4,31 +4,57 @@
...
     this.changeModalVisibility = this.changeModalVisibility.bind(this);
+    this.confirmedDeletionFromMyOutfit = this.confirmedDeletionFromMyOutfit.bind(this);
   }
 
   changeModalVisibility = () => {
-    const newVisibility = !this.state.showRelatedProductsModal;
+    let newVisibility;
 
-    this.setState({ showRelatedProductsModal: newVisibility });
+    if (this.props.carouselType === 'suggestions') {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx:25]: Separate The My Outfit Modal Options (Compare / My Outfit Delete)

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 17]
```diff
@@ -4,31 +4,57 @@
...
+      this.setState({ showSuggestionsModal: newVisibility });
+    }
+
+    if (this.props.carouselType === 'myOutfit') {
+      newVisibility = !this.state.showMyOutfitModal;
+      this.setState({ showMyOutfitModal: newVisibility });
+    }
+  }
+
+  confirmedDeletionFromMyOutfit = () => {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx:36]: Pass Up the Click Handler After Closing Modal

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 18]
```diff
@@ -4,31 +4,57 @@
...
-import ModalPopUp from '../ModalPopUp/ModalPopUp';
+import CompareModalPopUp from '../SuggestionsPopUp/SuggestionsModalPopUp';
+import MyOutfitModalPopUp from '../MyOutfitModalPupUp/MyOutfitModalPopUp';
 
 class ModalDashboard extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      showRelatedProductsModal: false,
+      showSuggestionsModal: false,
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx:15]: Separate The My Outfit Modal Options (Compare / My Outfit Delete)

File: src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx [hunk 19]
```diff
@@ -4,31 +4,57 @@
 /* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import './ModalDashboard.scss';
-import ModalPopUp from '../ModalPopUp/ModalPopUp';
+import CompareModalPopUp from '../SuggestionsPopUp/SuggestionsModalPopUp';
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/ModalDashboard/ModalDashboard.jsx]: Separate The My Outfit Modal Options (Compare / My Outfit Delete)

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.scss [hunk 20]
```diff
@@ -1,19 +1,13 @@
-.card-image img{
+.rp-card-image img{
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.scss [hunk 21]
```diff
@@ -1,4 +1,4 @@
-.product-card-main {
+.rp-product-card-main {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/OutfitCard/OutfitCard.scss [hunk 22]
```diff
@@ -1,4 +1,4 @@
-.add-item {
+.rp-add-item-button {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/OutfitCard/OutfitCard.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.scss [hunk 23]
```diff
@@ -1,12 +1,12 @@
-.carousel-main {
+.rp-carousel-main {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/EmptyCard/EmptyCard.scss [hunk 24]
```diff
@@ -1,4 +1,4 @@
-.empty-card {
+.rp-empty-card {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/EmptyCard/EmptyCard.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/Carousel/Carousel.scss [hunk 25]
```diff
@@ -1,13 +1,13 @@
-.carousel-main {
+.rp-carousel-main {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/Carousel/Carousel.scss:1]: Add 'rp-' Prefix to Privatize CSS Classes

File: public/index.html [hunk 26]
```diff
@@ -15,7 +15,7 @@
     
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     
-    <title>React Redux App</title>
+    <title>Tiger Eye</title>
```
Comment (thomHayner) [inline public/index.html:18]: Changed Page Title (The Name Displayed On Your Browser Tab)

File: src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx [hunk 27]
```diff
@@ -63,9 +62,17 @@ class Carousel extends Component {
     }
 
     return (
-      <div className="container-fluid carousel-main">
+      <div className="container-fluid rp-carousel-main">
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx:67]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/EmptyCard/EmptyCard.jsx [hunk 28]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './EmptyCard.scss';
 
 const EmptyCard = () => (
-  <div className="empty-card">
+  <div className="rp-empty-card">
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/EmptyCard/EmptyCard.jsx:5]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 29]
```diff
@@ -11,15 +9,15 @@ class MyOutfitCarousel extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      myOutfit: [],
+      myOutfit: [9, 8, 7, 6, 5, 4, 3, 2],
       myOutfitCarouselIndex: 0,
     };
     this.next = this.next.bind(this);
-    this.addToOutfit = this.addToOutfit(this);
+    this.addToOutfit = this.addToOutfit.bind(this);
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:16]: Fix / Add Function Bindings

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 30]
```diff
@@ -29,21 +27,29 @@ class MyOutfitCarousel extends Component {
...
-    if (changeToCard < 0) { // alter this to be dynamic based on number of related products
+    if (changeToCard < 0) {
       changeToCard = max - 3;
     }
 
     this.setState({ myOutfitCarouselIndex: changeToCard });
   }
 
-  addToOutfit(id) {
+  addToOutfit() {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:40]: Fix  addToOufit() Functionality

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 31]
```diff
@@ -29,21 +27,29 @@ class MyOutfitCarousel extends Component {
...
     const newOutfit = this.state.myOutfit.slice();
 
-    newOutfit.unshift(id);
-    console.log(newOutfit);
+    newOutfit.unshift(this.props.currentProduct);
+
+    this.setState({ myOutfit: newOutfit });
+  }
+
+  removeFromOutfit() {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:48]: Add addToOufit() Functionality

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 32]
```diff
@@ -61,9 +67,17 @@ class MyOutfitCarousel extends Component {
     }
 
     return (
-      <div className="container-fluid carousel-main">
+      <div className="container-fluid rp-carousel-main">
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:70]: Add 'rp-' Prefix to Privatize CSS Classes

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 33]
```diff
@@ -1,29 +1,75 @@
...
+        className="rp-card-image"
+        onClick={() => {}}
+        onKeyPress={() => { /* if {props.carouselType} === suggested, then redirect entire page to this card's product id */ }}
+        role="button"
+        tabIndex={0}
+      >
+        <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:29]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 34]
```diff
@@ -1,29 +1,75 @@
...
+          className="rp-button-overlay"
+          onClick={() => { /* if {props.carouselType} === suggested, then open modal; if {props.carouselType} === myOutfit, then remove from myOutfit */ }}
+          onKeyPress={() => {}}
+          role="button"
+          tabIndex={0}
+        >
+          <ModalDashboard
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:37]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 35]
```diff
@@ -1,29 +1,75 @@
...
+      role="button"
+      tabIndex={0}
+    >
+      <div
+        className="rp-button-overlay"
+        onClick={() => { /* if {props.carouselType} === suggested, then redirect entire page to this card's product id */ }}
+        onKeyPress={() => {}}
+        role="button"
+        tabIndex={0}
+      >
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:65]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 36]
```diff
@@ -1,29 +1,75 @@
...
-      <button type="button" className="button-overlay">+</button>
-      <img src={imageUrl} alt="ALT_IMAGE_TEXT_HERE" />
+    <div
+      className="rp-card-image"
+      onClick={() => { /* if {props.carouselType} === suggested, then redirect entire page to this card's product id */ }}
+      onKeyPress={() => {}}
+      role="button"
+      tabIndex={0}
+    >
+      <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:58]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx [hunk 37]
```diff
@@ -1,29 +1,75 @@
...
+        </div>
+        <img src="https://www.warnersstellian.com/Content/images/product_image_not_available.png" alt="not available" />
       </div>
     );
   }
   return (
-    <div className="card-image" onClick={() => {}} onKeyPress={() => {}} role="button" tabIndex={0}>
-      <button type="button" className="button-overlay">+</button>
-      <img src={imageUrl} alt="ALT_IMAGE_TEXT_HERE" />
+    <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCardImage/ProductCardImage.jsx:51]: Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx [hunk 38]
```diff
@@ -1,17 +1,19 @@
...
 import React, { useState, useEffect } from 'react';
 import './ProductCard.scss';
 // import { connect } from 'react-redux';
 import EmptyCard from '../EmptyCard/EmptyCard';
 import ProductCardImage from '../ProductCardImage/ProductCardImage';
 import Stars from '../../../../Shared/Stars';
+// import helperApi from '../../../../Shared/api';
 
-const ProductCard = (props) => {
+const ProductCard = ({
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx:9]: Destructuring

File: src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx (hunk 7 above)
Comment (thomHayner) [reply src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx:26]: A Conditional Render Was Removed, In Favor Of, A Dynamic Child Determined By Options Passed As Props

File: src/Components/RelatedProducts/RPComponents/modal/MyOutfitModalPupUp/MyOutfitModalPopUp.jsx [hunk 39]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/MyOutfitModalPupUp/MyOutfitModalPopUp.jsx:1]: Added A Second Type Of Modal Functionality / Pop Up

File: src/Components/RelatedProducts/RPComponents/modal/MyOutfitModalPupUp/MyOutfitModalPopUp.scss [hunk 40]
```diff
@@ -0,0 +1,28 @@
+.rp-modal {
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/MyOutfitModalPupUp/MyOutfitModalPopUp.scss]: Added A Second Type Of Modal Functionality / Pop Up

File: src/Components/RelatedProducts/RPComponents/modal/SuggestionsPopUp/SuggestionsModalPopUp.jsx [hunk 41]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/modal/SuggestionsPopUp/SuggestionsModalPopUp.jsx]: Added A Second Type Of Modal Functionality / Pop Up

File: src/Components/RelatedProducts/RPComponents/OutfitCard/OutfitCard.jsx [hunk 42]
```diff
@@ -1,12 +1,19 @@
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import './OutfitCard.scss';
 
 const OutfitCard = (props) => (
-  <button type="button" className="add-item" onClick={() => props.handleClick}>
+  <div
```
Comment (thomHayner) [inline src/Components/RelatedProducts/RPComponents/OutfitCard/OutfitCard.jsx:6]: Add 'rp-' Prefix to Privatize CSS Classes
Expanded for Readability, Code CleanUp

File: src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx [hunk 43]
```diff
@@ -23,13 +21,16 @@ class Carousel extends Component {
   }
 
   componentDidMount() {
-    fetch(`http://52.26.193.201:3000/products/${this.props.currentProduct}/related`)
+    const { mainProductId } = this.props;
```
Comment (morahan) [inline src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx:24]: is this pulling data from redux?

Comment (thomHayner) [reply src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx:24]: no this is fetching from the server direct to this component

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx [hunk 44]
```diff
@@ -11,15 +9,15 @@ class MyOutfitCarousel extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      myOutfit: [],
+      myOutfit: [9, 8, 7, 6, 5, 4, 3, 2],
```
Comment (morahan) [inline src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:12]: is this a default outfit?

**Review by morahan — APPROVED** [review]
Lots of changes to RelatedProducts component

File: src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx (hunk 44 above)
Comment (thomHayner) [reply src/Components/RelatedProducts/RPComponents/MyOutfitCarousel/MyOutfitCarousel.jsx:12]: yeah this is hard coded for now

**Pull request merged**